### DISCUSSION TITLE
How to enable a stylistic set for pandoc

### DISCUSSION THREAD

**Discussion by habere-et-dispertire** [tonsky/FiraCode#1430] [Q&A]
I use `pandoc --defaults=...`  to configure `pandoc`. I can't seem to get the `YAML` syntax right to enable a stylistic set, eg `ss02`.  The [Latex instructions](https://github.com/tonsky/FiraCode/wiki/LaTeX-instructions) are not enough. I tried:

~~~
  mainfontoptions:
  - StylisticAlternates=ss02
~~~

and

~~~
  mainfontoptions:
  - RawFeature=+ss02
~~~

and gave up after:

~~~
  header-includes:
  - \directlua{fonts.handlers.otf.addfeature{name="salt",type = "alternate",data={Q="ss02",},}}
~~~

**Comment by habere-et-dispertire** [comment]
The RawFeature option works but you need to make sure to apply it to the right font:

~~~
monofont: Fira Code
monofontoptions:
- RawFeature=+ss02
~~~

**Comment by MuTsunTsai** [comment]
You may also try https://mutsuntsai.github.io/fontfreeze/ to enable `ss02` permanently.

**Comment by jonassmedegaard** [comment]
The correct syntax is this:

```yaml
  mainfontoptions:
  - StylisticSet=02
```
